### PR TITLE
757 lint format text

### PR DESCRIPTION
**Pull request by ShiriMoran**
This PR finalizes aspects of wording and format of the lint warning/errors description and text

### CODE REVIEW

File: pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint [hunk 1]
```diff
@@ -1,47 +1,50 @@
...
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In VPC test-vpc1-ky, security group sg1-ky rule's indexed 1 splits subnet subnet1-ky (10.240.10.0/24). Splitting rule details: index: 1, direction: inbound,  conns: protocol: all, remote: sg1-ky (10.240.10.4/32), local: 0.0.0.0/0
-In VPC test-vpc1-ky, security group sg1-ky rule's indexed 3 splits subnets subnet2-ky (10.240.20.0/24), subnet3-ky (10.240.30.0/24). Splitting rule details: index: 3, direction: inbound,  conns: protocol: all, remote: sg2-ky (10.240.20.4/32,10.240.30.4/32), local: 0.0.0.0/0
-In VPC test-vpc1-ky, security group sg1-ky rule's indexed 4 splits subnet subnet3-ky (10.240.30.0/24). Splitting rule details: index: 4, direction: inbound,  conns: protocol: all, remote: sg3-ky (10.240.30.5/32,10.240.30.6/32), local: 0.0.0.0/0
+"SGs implying different connectivity for endpoints inside a subnet" issues:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In VPC "test-vpc1-ky", security group "sg1-ky" rule [1] splits subnet "subnet1-ky" (10.240.10.0/24).
```
Comment (kyorav) [inline pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint:22]: replace `Rule's details` with `Rule details`

File: pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
+In the connection from "test-vpc1-ky/vsi1-ky[10.240.10.4]" to "Public Internet 161.26.0.0/16" TCP response is blocked
+In the connection from "test-vpc1-ky/vsi2-ky[10.240.20.4]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP response is blocked
 ... (3 more)
 
 ________________________________________________________________________________________________________________________________________________________________________________________________________
 
 "NACL for which there are no resources attached to" issues:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-network ACL demilune-humorless-captain-lurex of VPC test-vpc1-ky has no resources attached to it
+network acl "demilune-humorless-captain-lurex" of VPC "test-vpc1-ky" has no resources attached to it
```
Comment (kyorav) [inline pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint:12]: same comment as above: make it "In the VPC "test-vpc1-ky", ..."

File: pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint [hunk 3]
```diff
@@ -1,32 +1,34 @@
...
+In VPC "test-vpc1-ky", network ACL "acl3-ky" rule [3] splits subnet "subnet3-ky" (10.240.30.0/24).
+	Rule's details: index: 3, direction: inbound , src: 10.240.20.0/24 , dst: 10.240.30.0/31, conn: all, action: allow
 ________________________________________________________________________________________________________________________________________________________________________________________________________
 
-"rules of network ACLs implying different connectivity for different endpoints within a subnet" issues:
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 1 splits subnet subnet3-ky (10.240.30.0/24). Splitting rule details: index: 1, direction: outbound , src: 10.240.30.0/31 , dst: 10.240.20.0/24, conn: all, action: allow
-In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 3 splits subnet subnet3-ky (10.240.30.0/24). Splitting rule details: index: 3, direction: inbound , src: 10.240.20.0/24 , dst: 10.240.30.0/31, conn: all, action: allow
+"SG for which there are no resources attached to" issues:
```
Comment (kyorav) [inline pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint:23]: I think this phrasing is mine, but I'm reading it again it is clunky. How about

"Security Groups not applied to any resources"

File: pkg/ibmvpc/examples/out/lint_out/acl3_shadowed_rules_Lint [hunk 4]
```diff
@@ -1,43 +1,46 @@
...
+In the connection from "test-vpc1-ky/vsi2-ky[10.240.20.4]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP response is blocked
+In the connection from "test-vpc1-ky/vsi3a-ky[10.240.30.5]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP response is blocked
 ... (2 more)
 
 ________________________________________________________________________________________________________________________________________________________________________________________________________
 
 "NACL for which there are no resources attached to" issues:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-network ACL demilune-humorless-captain-lurex of VPC test-vpc1-ky has no resources attached to it
+network acl "demilune-humorless-captain-lurex" of VPC "test-vpc1-ky" has no resources attached to it
```
Comment (kyorav) [inline pkg/ibmvpc/examples/out/lint_out/acl3_shadowed_rules_Lint:12]: same as above

File: pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint [hunk 5]
```diff
@@ -1,47 +1,50 @@
...
+In the connection from "test-vpc1-ky/vsi2-ky[10.240.20.4]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP src-ports: 1-99,201-65535; TCP src-ports: 100-200 dst-ports: 51-65535 response is blocked
+In the connection from "test-vpc1-ky/vsi3a-ky[10.240.30.5]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP src-ports: 1-99,201-65535; TCP src-ports: 100-200 dst-ports: 1-9,51-65535 response is blocked
 ... (2 more)
 
 ________________________________________________________________________________________________________________________________________________________________________________________________________
 
 "NACL for which there are no resources attached to" issues:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-network ACL corrode-kilogram-cola-mandated of VPC test-vpc1-ky has no resources attached to it
+network acl "corrode-kilogram-cola-mandated" of VPC "test-vpc1-ky" has no resources attached to it
```
Comment (kyorav) [inline pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint:12]: Why are we changing "ACL" to "acl"? I think the acronym should stay capitalized (everywhere, not just here).

File: pkg/ibmvpc/examples/out/lint_out/multivpc_Lint [hunk 6]
```diff
@@ -1,42 +1,42 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+VPC "test-vpc2-ky"'s subnet "subnet21-ky" [10.240.64.0/24] and VPC "zn-vpc2"'s subnet "zn-vpc2-net1" [10.240.64.0/24] overlap
 ________________________________________________________________________________________________________________________________________________________________________________________________________
 
 "SG for which there are no resources attached to" issues:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-security group disrupt-stem-mulch-moneybags of VPC zn-vpc2 has no resources attached to it
-security group heroics-diffused-book-estranged of VPC test-vpc2-ky has no resources attached to it
-security group relenting-sixfold-moisturize-emcee of VPC test-vpc0-ky has no resources attached to it
+security group "disrupt-stem-mulch-moneybags" of VPC "zn-vpc2" has no resources attached to it
```
Comment (kyorav) [inline pkg/ibmvpc/examples/out/lint_out/multivpc_Lint:23]: Same as above:
```
In the VPC "zn-vpc2", the security group ...
```

**Review by kyorav — CHANGES_REQUESTED** [review]

File: pkg/ibmvpc/examples/out/lint_out/acl3_shadowed_rules_Lint [hunk 7]
```diff
@@ -1,43 +1,46 @@
...
 "rules of network ACLs that are shadowed by higher priority rules" issues:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In VPC test-vpc1-ky network ACL acl2-ky's rule 6 is redundant. It is shadowed by higher priority rules
-	Rule's details: index: 6, direction: outbound , src: 10.240.20.0/28 , dst: 10.240.10.0/24, conn: all, action: allow
+In VPC "test-vpc1-ky", network ACL "acl2-ky" rule [6] is shadowed by higher priority rules
+	Rule details: index: 6, direction: outbound , src: 10.240.20.0/28 , dst: 10.240.10.0/24, conn: all, action: allow
 		Shadowing rules:
 		index: 1, direction: outbound , src: 10.240.20.0/24 , dst: 10.240.10.0/24, conn: protocol: icmp, action: allow
-		index: 2, direction: outbound , src: 10.240.20.0/24 , dst: 10.240.10.0/24, conn: all, action: allow
+			index: 2, direction: outbound , src: 10.240.20.0/24 , dst: 10.240.10.0/24, conn: all, action: allow
```
[the remark this answers is not in the document]
Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/acl3_shadowed_rules_Lint:36]: oops
this last change before the push ;-)

good catch, fixed

File: pkg/linter/lintSubnetCIDROverlap.go [hunk 8]
```diff
@@ -24,7 +24,7 @@ func newSubnetCIDROverlap(name string, configs map[string]*vpcmodel.VPCConfig,
 		basicLinter: basicLinter{
 			configs:     configs,
 			name:        name,
-			description: "Overlapping CIDR ranges between different subnets",
+			description: "Overlapping Subnet address spaces",
```
[the remark this answers is not in the document]
Comment (ShiriMoran) [reply pkg/linter/lintSubnetCIDROverlap.go:27]: done

File: pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint (hunk 5 above)
[the remark this answers is not in the document]
Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint:12]: But here's the thing: there could be more than one `VPC` - if the `src` and `dst` are not in the same `VPC`.  I think its more important that all issues of the same lint have the same format than that issues of different lints do.

About mentioning or not mentioning the `VPC` - not sure. Perhaps mention only if the config has more than a single `VPC`? 
Or just leave as it is now and wait for "real" feedback

File: pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint (hunk 1 above)
Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint:22]: done

File: pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint (hunk 5 above)
[the remark this answers is not in the document]
Comment (kyorav) [reply pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint]: I don't understand this comment. The lint is about a network ACL that is not applied to any subnet -- how does that relate to source and destination?

File: pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint (hunk 3 above)
Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint:23]: also "NACL not applied to any resources" ?
@zivnevo agreed?

File: pkg/ibmvpc/examples/out/lint_out/acl3_shadowed_rules_Lint (hunk 4 above)
Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/acl3_shadowed_rules_Lint]: done

File: pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint (hunk 5 above)
Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint]: sure. somehow `toLower` got into my code ;-)

File: pkg/ibmvpc/examples/out/lint_out/multivpc_Lint (hunk 6 above)
Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/multivpc_Lint]: done

File: pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint (hunk 5 above)
[the remark this answers is not in the document]
Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint]: I misread. already fixed.

File: pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint (hunk 3 above)
Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint:23]: @zivnevo ? (`unattached` was agreed today with him)

Comment (ShiriMoran) [reply pkg/ibmvpc/examples/out/lint_out/acl3_3rd_Lint:23]: fixed for both

**Review by zivnevo — APPROVED** [review]

**Review by kyorav — APPROVED** [review]

**Pull request merged**